### PR TITLE
Fixed some bugs

### PR DESCRIPTION
**Pull request by DailisLangovskis**

### CODE REVIEW

File: css/app.scss [hunk 1]
```diff
@@ -546,6 +546,9 @@ button.but-title-sm {
 	position: relative;
 	font-family: Roboto, "Helvetica Neue", sans-serif;
 }
+.hs-ows-wms-capabilities-error {
```
Comment (jmacura) [inline css/app.scss:549]: Not really a fix, but a workaround, right? The font-family shall be inherited from .hs-content-wrapper to all the dialogs, which is not..

**Comment by jmacura** [comment]
[Release 2.5.0](https://trello.com/c/ve259N2U/132-release-250)

File: css/app.scss (hunk 1 above)
Comment (DailisLangovskis) [reply css/app.scss:549]: Well, yeah, I don't know how to do it in other way, but at least made it, so it works for now.

**Review by jmacura — APPROVED** [review]
🐛 🔫

**Pull request merged**